src/shared/components/palette/collection/write-collection-field.html

```diff
@@ -40,9 +44,9 @@ <h2 class="heading-h2 error-spacing">
       </div>
 
     </div>
-
-    <button class="button" type="button" (click)="addItem()" [disabled]="isNotAuthorisedToCreate()" *ngIf="caseField.value && caseField.value.length">Add new</button>
-
+    <ng-container *ngIf="!isNotAuthorisedToCreate()">
```

Comment:
double negation - one from "!" and another from the function name
makes code less smooth to read